docs/index.md

```diff
@@ -69,71 +80,55 @@ The easiest way to get started with Swarm is to use the
 $ docker pull swarm
 ```
 
-## Set up Swarm nodes
+## Set up Swarm
 
-Each Swarm node will run a Swarm node agent. The agent registers the referenced
-Docker daemon, monitors it, and updates the discovery backend with the node's status.
+1. Log into **each machine** and do the following:
 
-The following example uses the Docker Hub based `token` discovery service:
-
-1. Create a Swarm cluster using the `docker` command.
+    Start the Docker daemon with the `-H` flag. This ensures that the Docker remote API
+    is available over TCP to the *Swarm Manager*.
 
     ```bash
-    $ docker run --rm swarm create
-    6856663cdefdec325839a4b7e1de38e8 # <- this is your unique <cluster_id>
+	$ docker -H tcp://0.0.0.0:2375 -d
```

Comment:
I wonder if the example should show how to set up a non-secure API. Even though the warning is added above, users will still blindly follow the guide and end up with an insecure set-up.

/cc @ewindisch @diogomonica